platform/core/src/utils/index.test.js

```diff
@@ -7,6 +7,7 @@ describe('Top level exports', () => {
       'ObjectPath',
       'absoluteUrl',
       'addServers',
+      'addUserPreferences',
```

Comment:
Should this be "add" or "set"?

Reply:
Just remove this file, after some refactors its not necessary anymore.